tests/unit/ember-sinon-qunit-test.js

```diff
@@ -56,3 +56,15 @@ test('sinon sandbox cleans up after itself', function (assert) {
   assert.ok(!spy.called, 'spy has no registered calls');
   assert.ok(!stub.called, 'stub has no registered calls');
 });
+
+test('does not fake timers by default', function (assert) {
+  assert.notEqual(Date.now(), 0, 'Date.now() is not reset');
+});
+
+test('does fake timers on request', { useFakeTimers: true }, function (assert) {
```

Comment:
I'm not a huge fan of completely changing the signature of the `test` method, since I've striven to make it look and feel exactly like the `test` method you get from `ember-qunit` by default. I don't think it's particularly unusable to use `beforeEach`/`afterEach` to change the `fakeTimers` setting for a whole block of tests in a file, but I'd welcome more input on this point. I personally almost never have need to use fake timers in the first place, so I don't have as much experience with it.